src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2465,8 +2465,26 @@ def create_managed_ssl_cert(cmd, resource_group_name, name, hostname, slot=None)
     location = webapp.location
     easy_cert_def = Certificate(location=location, canonical_name=hostname,
                                 server_farm_id=server_farm_id, password='')
-    return client.certificates.create_or_update(name=hostname, resource_group_name=resource_group_name,
-                                                certificate_envelope=easy_cert_def)
+
+    try:
+        return client.certificates.create_or_update(name=hostname, resource_group_name=resource_group_name,
+                                                    certificate_envelope=easy_cert_def)
+    except DefaultErrorResponseException as ex:
+        if ex.response.status_code == 202:
+            retry_times = 12
```

Comment:
Do we need the re-try lmit I was hoping CLI/client will have a threshold for time out otherwise polling on the operation status using GET should return 200 in <2 mins or non apex domains in my testing with portal for the same

Reply:
Updated to use 2 minute timeout